app/views/details_not_found.scala.html

```diff
@@ -51,6 +51,9 @@ <h2 class="heading-small" id="details-not-found-name-changed-reason-header">@mes
   <h2 class="heading-small" id="details-not-found-utr-reason-header">@messages("bc.details-not-found.utr-reason.header")</h2>
   <p id="details-not-found-utr-reason-text">@messages("bc.details-not-found.utr-reason.text")</p>
 
-  <a class="button" role="button" data-journey-click="business-customer:click:try-matching-again" href="@controllers.routes.BusinessVerificationController.businessForm(service, businessType)">@messages("bc.details-not-found.try-again")</a>
+
+@form(action=controllers.routes.BusinessVerificationController.businessForm(service, businessType)){
+<button class="button" role="button" data-journey-click="business-customer:click:try-matching-again" >@messages("bc.details-not-found.try-again")</button>
```

Comment:
We probably need to remove all instances of role="button" as it is observed in the VNU log https://build.tax.service.gov.uk/job/DDCT%20Live%20Services/job/Business%20Customer%20Services/job/business-customer-accessibility-tests/2/Accessibility_20Check_20Report/

Reply:
I will make all the changes and run the tests again